index.rb

```diff
@@ -66,6 +66,25 @@ def show(**kwargs)
   end
 end
 
+def seetickets_parser(event)
```

Comment:
Now that I think about it, maybe this should be a method on `Venue`, like `seeticket_show` that takes a single nokogiri element and returns a `Show`.